Dockerfile.image

```diff
@@ -1,6 +1,7 @@
 # ARGS go first if used on FROM
 ARG OPERATOR_IMAGE=quay.io/costoolkit/elemental-operator:latest
 ARG SYSTEM_AGENT_IMAGE=rancher/system-agent:v0.2.9
+ARG TOOL_IMAGE=quay.io/costoolkit/elemental-cli:v0.0.15-87f0cb4
```

Comment:
@Itxaka  I just have a doubt about how you envision to maintain this, I mean, I assume we simply manually update this value as we need, but is there a some automation to ensure it follows the latest version? or would it make sense to have also publish `quay.io/costoolkit/elemental-cli:latest` so we don't have to worry to pin some specific commit unless we explicitly need to test some specific version.

Just wondering about how this should be handled, any suggestion? In any case like having the option to build locally a built of my choice.

Reply:
yep, I would like to move this(elemental-cli) and the operator to publish their master images into the `REPO-ci` so we can use those for CI/CD  easily by setting this to that latest tag.

Currently elemental-cli pushes master AND tag versions to the "release" repo which it was good before when it was a moving target but its not needed anymore. Idea would be like operator which pushes master+latest ot the quay.io/costoolkit/elemental-operator-ci and tags+latest to quay.io/costoolkit/elemental-operator repo as to have a clear separation between what is stable and what is dev

Once that is done, this can be moved to track the ci latest tag so we cant test with latest of everything in github and have a good idea if we break something.